Detections/SigninLogs/BypassCondAccessRule.txt

```diff
@@ -0,0 +1,23 @@
+// Name: Attempt to Bypass conditional access rule
+//
+// Description: This query over Azure AD sign-in activity highlights attempt to Bypass conditional access rule. The  ConditionalAccessStatus column value in the logs details if there was an attempt to bypass Conditional Access or if the Conditional access rule was not satisfied (ConditionalAccessStatus == 1 )
```

Comment:
if it's a detection then it needs info like: 
// QueryFrequency: 24
//
// QueryPeriod: 24
//
// AlertTriggerOperator: gt
//
// AlertTriggerThreshold: 0

Reply:
Made some additional changes, will push shortly.